src/video_core/renderer_vulkan/vk_presenter.cpp

```diff
@@ -636,8 +636,22 @@ void Presenter::Present(Frame* frame) {
 
     if (!swapchain.AcquireNextImage()) {
         swapchain.Recreate(window.GetWidth(), window.GetHeight());
+        if (!swapchain.AcquireNextImage()) {
+            // User resizes the window too fast and GPU can't keep up. Skip this frame.
+            LOG_WARNING(Render_Vulkan, "Skipping frame!");
+            // Free the frame for reuse
+            std::scoped_lock fl{free_mutex};
```

Comment:
Small nit, could you make this into a lambda defined in this function so it can be used both here and below

Reply:
Done.